listenbrainz/webserver/views/entity_pages.py

```diff
@@ -209,14 +236,54 @@ def artist_entity(artist_mbid):
 
 
 @album_bp.route("/",  defaults={'path': ''})
-@album_bp.route('/<path:path>/')
-def album_page(path):
-    return render_template("index.html")
+@album_bp.route("/<release_group_mbid>/", methods=["GET"])
+def album_page(release_group_mbid: str):
+    og_meta_tags = None
+    if is_valid_uuid(release_group_mbid):
+        metadata = fetch_release_group_metadata(
+            [release_group_mbid],
+            ["artist", "recording"]
+        )
+        if len(metadata) == 0:
+            pass
+        else:
+            release_group = metadata[release_group_mbid]
+
+            recording_data = release_group.pop("recording")
+            mediums = recording_data.get("mediums", [])
+            recording_mbids = []
+            for medium in mediums:
+                for track in medium["tracks"]:
+                    recording_mbids.append(track["recording_mbid"])
+
+            album_name = release_group.get("release_group").get("name")
+            artist_name = release_group.get("artist").get("name")
+            track_count = len(recording_mbids)
+            listening_stats = get_entity_listener(
+                db_conn, "release_groups", release_group_mbid, "all_time")
+            total_listen_count = 0
+            if listening_stats and "total_listen_count" in listening_stats:
+                total_listen_count = number_readable(
+                    listening_stats["total_listen_count"] or 0)
+
+            og_meta_tags = {
+                "title": f'{album_name} — {artist_name}',
+                "description": f'Album — {track_count} tracks — {total_listen_count} listens — ListenBrainz',
+                "type": "music.album",
+                "music:musician": artist_name,
+                "music:release_date": release_group.get("release_group").get("date"),
+                "image": f'https://coverartarchive.org/release-group/{release_group_mbid}/front-500',
+                "image:width": "500",
+                "image:alt": f"Cover art for {album_name}",
+                "url": f'{current_app.config["SERVER_ROOT_URL"]}/album/{release_group_mbid}',
```

Comment:
Same as above. 

React Helmet renders the title as `album_name - ListenBrainz`.